### PR TITLE
XKCD Chart Easter Egg

### PR DESCRIPTION
**Pull request by tlrobinson**

### CODE REVIEW

File: src/metabase/models/card.clj [hunk 1]
```diff
@@ -18,7 +18,8 @@
     :scalar
     :state
     :table
-    :timeseries})
+    :timeseries
+    :xkcd})
```
Comment (agilliland) [inline src/metabase/models/card.clj]: i don't believe there is any reason to have this here.  i reverted this locally and everything works fine.

**Comment by agilliland** [comment]
I'm seeing this in my console.  doesn't seem to be causing any problems, but ideally it wouldn't be happening.

screen shot 2016-03-21 at 3 26 44 pm

**Comment by agilliland** [comment]
@tlrobinson can you take one final look at the last couple comments i left here?  after that i think this should be g2g.

**Comment by tlrobinson** [comment]
Fixed the font errors, and thus fonts.

**Pull request merged**